README.md

```diff
@@ -1,2 +1,363 @@
-# learn-payment-processing
-[WiP] Learn how to process online payments in your web application! :money:
+<div align="center">
+
+# Learn Payment Processing
+
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/dwyl/learn-payment-processing/ci.yml?label=build&style=flat-square&branch=main)
+[![codecov.io](https://img.shields.io/codecov/c/github/dwyl/learn-payment-processing/main.svg?style=flat-square)](https://codecov.io/github/dwyl/phoenix-chat-example?branch=main)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat-square)](https://github.com/dwyl/phoenix-chat-example/issues)  
+[![HitCount](https://hits.dwyl.com/dwyl/learn-payment-processing.svg?style=flat-square&show=unique)](http://hits.dwyl.com/dwyl/learn-payment-processing)
+  
+Learn what payment processing is 
+and how you can add it to your application!
+
+</div>
+
+# Why? 🤷
+
+Not all applications are free.
+There are some that can be acquired through
+a [one time purchase](https://git-fork.com/buy).
+There are others
+that are [subscription-based](https://www.notion.so/pricing).
+Regardless of the type, all of these share one thing in common:
+**they use payment processing platforms/gateways
+to manage transactions**.
+
+In any current application, 
+knowing how processing payments *work*
+and how they *can* be implemented 
+is important, 
+as it affects the bottom-line
+of the project/application/company. 
+
+# What 💭
+
+Let's think for a minute of what happens
+when we purchase something online.
+Assume we want to [buy Fork](https://git-fork.com/buy), 
+a neat open-source Git GUI. 
+
+<img width="1198" alt="buy-fork-1" src="https://user-images.githubusercontent.com/17494745/208933958-8747bf88-cd74-4ec1-8739-430fdb1874ab.png">
+
+If we click the `Buy Fork` button,
+we are met with a modal 
+for a one-time purchase license.
```

Comment:
This is not a good example. 
See: https://github.com/dwyl/product-ux-research/issues/38#issuecomment-1364001758
We do not want to promote this website because the UX is _horrible_.

A _much_ better but still not great example of a one-time payment for software is: https://eggerapps.at/postico2/
https://user-images.githubusercontent.com/194400/209484028-fa51dc93-7fd9-495a-9ab5-68682c955cfc.mov

Clear transition between screens and _single_ page checkout that clearly displays total price.

Created **`#research`** task on backlog to dive into this a bit deeper: https://github.com/dwyl/product-ux-research/issues/44